.claude/skills/github/scripts/pr/new_pr.py

```
@@ -133,10 +137,10 @@ def run_validations(
     # Validation 3: Test coverage detection (WARNING)
     print()
     print("[3/4] Checking test coverage...")
-    test_script = os.path.join(repo_root, "scripts/Detect-TestCoverageGaps.ps1")
+    test_script = os.path.join(repo_root, "scripts/detect_test_coverage_gaps.py")
     if os.path.exists(test_script):
         subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)